test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Output/AttributeTargetingTagHelpers.DesignTime.cs

```diff
@@ -20,10 +20,10 @@ private void @__RazorDesignTimeHelpers__()
             #pragma warning restore 219
         }
         #line hidden
-        private PTagHelper __PTagHelper = null;
-        private CatchAllTagHelper __CatchAllTagHelper = null;
-        private InputTagHelper __InputTagHelper = null;
-        private InputTagHelper2 __InputTagHelper2 = null;
+        private global::PTagHelper __PTagHelper = null;
```

Comment:
I mean `global::PTagHelper` and so on implies a bug in the code. Where are the namespaces for the tag helper classes? Since you're crafting the `TagHelperDescriptor`s manually, it's a test issue. So fix it at the source.